packages/orchestration/src/orchestration-response.ts

```diff
@@ -21,7 +21,7 @@ export class OrchestrationResponse {
    * @returns Token usage.
    */
   getTokenUsage(): TokenUsage {
-    return this.data.orchestration_result.usage;
+    return this.data.orchestration_result.usage!;
```

Comment:
This is fine to do because this is only undefined in case of streaming. To be fixed in the spec